bin/nyc.js

```diff
@@ -50,7 +51,7 @@ if (argv._[0] === 'report') {
     NYC_CONFIG: JSON.stringify(config),
     NYC_CWD: process.cwd(),
     NYC_ROOT_ID: nyc.rootId,
-    BABEL_DISABLE_CACHE: 1,
+    BABEL_DISABLE_CACHE: config['babel-cache'],
```

Comment:
I'd be tempted to do:

```js
if (config['babel-cache'] === false) {
  // the babel cache in few cases does not play nicely with nyc.
  process.env.BABEL_DISABLE_CACHE = '1'
} else {
  process.env.BABEL_DISABLE_CACHE = '0'
}
```

followed by:

`BABEL_DISABLE_CACHE: process.env.BABEL_DISABLE_CACHE`

rather than:

`BABEL_DISABLE_CACHE: config['babel-cache']`

Definitely make sure this approach works on the repo that you've been benchmarking 👍